app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -81,12 +82,40 @@ export const ScanningQRCodeScreen: ScreenType = ({
     }
 
     try {
-      const { valid } = validPayment(data, tokenNetwork, myPubKey, username)
+      const { valid, lnurl } = validPayment(data, tokenNetwork, myPubKey, username)
       if (valid) {
-        if (index <= 1) {
-          navigation.replace("sendBitcoin", { payment: data })
+        if (lnurl) {
+          setPending(true)
+          const lnurlParams = await getParams(lnurl)
+
+          if ("reason" in lnurlParams) {
+            throw lnurlParams.reason
+          }
+
+          switch (lnurlParams.tag) {
+            case "withdrawRequest":
+              // TODO: lnurl-withdraw
```

Comment:
thinking about the UX of this, would this cause a flow where the user tries to withdraw and just can't, but there is no message explaining anything? if that's the case, we should just add some kind of message like "lnurl withdrawals not supported yet"

Reply:
Great feedback, thanks @FuzzWorley, I've added user friendly error messaging here.